lib/-ftb-fzf

```diff
@@ -64,6 +64,9 @@ fi
 
 cat > $tmp_dir/completions.$$
 
+
+_ftb_query="${_ftb_query}$(command gdd bs=1G count=1 status=none iflag=nonblock < /dev/tty 2>/dev/null)" || true
```

Comment:
I think this only works for macOS users, in Linux there is only `dd`.

You can use `$OSTYPE` to determine which one should be used here.

Reply:
@Aloxaf Ok, I added some checks which should be sufficient.